homeassistant/components/zwave.py

```diff
@@ -17,6 +20,14 @@
     EVENT_HOMEASSISTANT_STOP, EVENT_PLATFORM_DISCOVERED)
 from homeassistant.util import convert, slugify
 
+# Try to set a sane default if libopenzwave is installed
+try:
+    DEFAULT_ZWAVE_CONFIG_PATH = os.path.join(os.path.dirname(
+        libopenzwave.__file__), 'config')
+except NameError:
+    DEFAULT_ZWAVE_CONFIG_PATH = os.path.join(sys.prefix, 'share',
```

Comment:
The idea is to default to `libopenzwave.__file__` as the config location if one is not specified. This will be the correct location except in edge cases where the user has modified/updated the libopenzwave config manually. The goal is for elimiating `config_path: ` as a required yaml entry except for those rare edge cases.

You're right about the failure mode however, if libopenzwave is not found, we should exit the component then and there.